stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/attachments/images/ImagesPicker.kt

```diff
@@ -121,5 +141,35 @@ internal fun DefaultImagesPickerItem(
                 )
             }
         }
+
+        if (isVideo) {
+            Icon(
+                modifier = Modifier
+                    .padding(4.dp)
+                    .size(24.dp)
+                    .mirrorRtl(LocalLayoutDirection.current)
+                    .align(Alignment.BottomStart),
+                painter = painterResource(id = R.drawable.stream_compose_ic_video),
+                contentDescription = null,
+                tint = Color.White
```

Comment:
What happens if we have a fully white video? or a white thumbnail? We won't be able to see the icon or the text.

Can we add a dark gradient or something to the bottom or alternatively add a small shape to the icon and the text? Or have the text and the icon one next to the other in a Row and then add a shape?